### PR TITLE
[ZH] Add simplified Chinese

### PR DESCRIPTION
**Pull request by Jesse205**
# Description

**This pull request solves/implements:** #123

# Imagery translation

| Original text        | Translation |
| -------------------- | ----------- |
| Quick Settings Tile  |   快速设置磁贴          |
| Home Screen Widgets  |    主屏幕小部件         |
| Widget Customization |     自定义小部件        |
| Instant Connection   |   即时连接          |
| OneUI Integration |OneUI 集成|
| Root or Shizuku| Root 或者 Shizuku|
# Rules and agreements

- [x] My commit messages comply with [Conventional commits](https://www.conventionalcommits.org/) (use your IDE's Git manager to edit the commit messages if they don't)
- [x] Original sources of code and imagery (if any) are properly credited
- [x] I agree to keep the discussion under this pull request polite and on point

- [x] I have read [translate.md](translate.md) and this pull request complies with it
- [x] This pull request does not translate [license.md](license.md) and [translate.md](translate.md)
- [x] My translations are of good quality and does not contain inappropriate content

### CODE REVIEW

**Comment by Jesse205** [comment]
For fonts, you can use HarmonyOS Sans SC, which is commercially available for free. But I suggest you read the license file before you use it.

https://developer.huawei.com/images/download/next/HarmonyOS-Sans.zip

**Comment by Smooth-E** [comment]
Thanks for being interested in the development of WADBS, @Jesse205!

I added the imagery with the text you provided in the initial table. What are your thoughts on it? Is it okay that most of the text is on one line?

I used [Roboto Mono](https://fonts.google.com/specimen/Roboto+Mono) for the font. Please mention it in the readme file where I left the comment. Use the link below:

```
https://fonts.google.com/specimen/Roboto+Mono
```

Also, shouldn't the licensing section in the readme file be translated as well? I mean, you should not translate the license file itself, but this small abstract-like section is better to be translated so Chinese-speaking users can get the gist of the license easily.

File: readme.zh_cn.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+它提供了几个小工具和一个快速设置磁贴来实现这一功能。 
+您还可以使用 KDE Connect 的剪贴板共享功能（参见 [scripts](./scripts)）即时连接到电脑。
+
+## 特性
+
+<img src="./media/feature-qs-tile.png" alt="快速设置磁贴"/>
+<img src="./media/feature-widgets.png" alt="主屏幕小工具"/>
+<img src="./media/feature-customization.png" alt="自定义小工具"/>
+<img src="./media/feature-instant-connection.png" alt="即时连接"/>
+<img src="./media/feature-shizuku.png" alt="与 Shizuku 工作"/>
```
Comment (Smooth-E) [inline readme.zh_cn.md]: The `alt` text here differs from the one you provided in the table and which is displayed on the picture itself. The text here translates as "working with Shizuku". This does not convey the idea that WADBS can also work with root. Please, replace the `alt` text with something like "works with root or Shizuku" or just "root or Shizuku".

**Comment by Jesse205** [comment]
> Thanks for being interested in the development of WADBS, @Jesse205!
> 
> I added the imagery with the text you provided in the initial table. What are your thoughts on it? Is it okay that most of the text is on one line?
> 
> I used [Roboto Mono](https://fonts.google.com/specimen/Roboto+Mono) for the font. Please mention it in the readme file where I left the comment. Use the link below:
> 
> ```
> https://fonts.google.com/specimen/Roboto+Mono
> ```
> 
> Also, shouldn't the licensing section in the readme file be translated as well? I mean, you should not translate the license file itself, but this small abstract-like section is better to be translated so Chinese-speaking users can get the gist of the license easily.

I'm looking at the pictures you committed. This font looks strange, probably because of some default font. Roboto Mono may not contain Chinese (Actually, I haven't seen a mono font containing Chinese.) The normal situation should be like this.
Screenshot_20240724_144928_cn wps moffice_i18n

**Comment by Smooth-E** [comment]
Alright, I updated the imagery to use HarmonyOS Sans. You can link one of these as the source, I think.

```
https://developer.huawei.com/consumer/cn/design/resource-V1/
```
```
https://github.com/huawei-fonts/HarmonyOS-Sans
```

**Comment by Jesse205** [comment]
Is it possible to have separate fonts for Chinese characters and English letters? If so, I suggest Roboto Mono for English letters and HarmonyOS Sans SC or Noto for Chinese characters (I found that Roboto font itself doesn't support Simplified Chinese either, sorry about that, Noto font on Android supports Simplified Chinese)

**Comment by Smooth-E** [comment]
> Is it possible to have separate fonts for Chinese characters and English letters?

Sure, why not. Here is a comparison:

| HarmonyOS Sans SC + HarmonyOS Sans | HarmonyOS Sans SC + Roboto Mono | HormonyOS Sans SC + Noto Sans |
| ---------------------------------- | ------------------------------- | ----------------------------- |
| image | image | image |

What do you think? In my opinion HarmonyOS fonts look better together.

**Comment by Smooth-E** [comment]
Examined the folder structure you created once more. Why did you name the folders in `app/src/main/res/` and `fastlane/metadata/android/` as `zh-rCN` and not just `zh-CN` or `zh`? What does the letter `r` mean there?

**Comment by Jesse205** [comment]
> Examined the folder structure you created once more. Why did you name the folders in `app/src/main/res/` and `fastlane/metadata/android/` as `zh-rCN` and not just `zh-CN` or `zh`? What does the letter `r` mean there?

In Android, Android Studio generated a folder for 'Chinese-China' as `zh-rCN` instead of the more commonly seen `zh-CN`. Similarly, I noticed pt-rBR in the metadata directory, so I considered adding an `r` to folder name as well.

**Comment by Smooth-E** [comment]
Person in F-Droid Matrix chat said that `r` prefixes for Portuguese Brazilian and Simplified Chinese are only used in Android directory names and not for fastlane metadata directories, so I renamed the folders.

**Comment by Smooth-E** [comment]
Thanks for your contribution!

**Comment by Smooth-E** [comment]
@Jesse205 if you are interested, you can also translate files in [scripts](https://github.com/Smooth-E/wireless-adb-switch/tree/main/scripts) and resources for the [widget-factory module](https://github.com/Smooth-E/wireless-adb-switch/blob/main/widget-factory/src/main/res/values/strings.xml). The latter is quite necessary as some of the strings are used on the main screen of the app.

**Review by zzzzzyyyyyyaaaaa-hub — APPROVED** [review]

**Review by zzzzzyyyyyyaaaaa-hub — APPROVED** [review]

**Pull request merged**